src/app/js/public/facet/FacetValueList.js

```diff
@@ -75,7 +78,7 @@ export const FacetValueList = ({
     invertFacet,
     sortFacetValue,
 }) => (
-    <div className="facet-value-list">
+    <div className="facet-value-list" style={styles.list}>
```

Comment:
it should be better to have only classnames, no style

Reply:
There is no classnames mechanism in this page (see the whole component).